static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -16,15 +16,15 @@ define([
         event.preventDefault();
         var questionElement = event.currentTarget.querySelector('.user__question-upvote');
         var atomIdElement = $('.js-storyquestion-atom-id');
-        
+
         if (questionElement && atomIdElement) {
             var questionId = questionElement.id;
             var atomId = atomIdElement.attr('id');
 
-            var question = $('#js-question-text-' + questionId);
+            var question = document.querySelector("meta[name=js-notranslate-" + questionId + "]");
```

Comment:
worth knowing: you can use an `id` attribute instead and query with `document.getElementById`, much faster. for a single pass, that is not necessary though.